play-services-core/src/main/kotlin/org/microg/gms/accountsettings/ui/WebViewHelper.kt

```diff
@@ -109,7 +111,13 @@ class WebViewHelper(private val activity: AppCompatActivity, private val webView
             }
             Log.d(TAG, "Opening $authUrl")
             webView.post {
-                loadWebViewUrl(authUrl)
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+                    CookieManager.getInstance().removeAllCookies {
+                        loadWebViewUrl(authUrl)
+                    }
+                } else {
+                    loadWebViewUrl(authUrl)
```

Comment:
Before Lollipop there is https://developer.android.com/reference/android/webkit/CookieManager#removeAllCookie()
(without the final s).
It seems similar but NOY asynchronous.